packages/postgres-database/src/simcore_postgres_database/utils_folders.py

```diff
@@ -415,6 +425,53 @@ def _get_permissions_where_clause() -> ColumnElement | bool:
 
 
 async def _check_folder_and_access(
+    connection: SAConnection,
+    product_name: _ProductName,
+    folder_id: _FolderID,
+    gids: set[_GroupID],
+    *,
+    permissions: _FolderPermissions,
+    enforece_all_permissions: bool,
```

Comment:
enforce

Reply:
Q: you are checking for access rights on a folder. the folder has persmissions. Why would you only have to enforce *some* of the permissions?

Reply:
Maybe the name could be better. The issue is you want to check if the permissions are set to "True" in the sense that you have them. You don't care about the "False" since some of the users can have False or True on that field.

As an example: to list the folders I don't care if you have "delete" set to True or False.